ovos_dinkum_listener/service.py

```diff
@@ -659,20 +659,38 @@ def _record_end_signal(self):
             )
         self.bus.emit(Message("recognizer_loop:record_end"))
 
-    def _stt_text(self, transcripts: List[Tuple[str, float]],
-                  stt_context: dict):
-        # Report utterance to intent service
-        if transcripts:
-            utts = [u[0] for u in transcripts]  # filter confidence
+    def __normtranscripts(self, transcripts: List[Tuple[str, float]]) -> List[str]:
+        # unfortunately common enough when using whisper to deserve a setting
+        # mainly happens on silent audio, not as a mistranscription
+        default_hallucinations = [
+            "thanks for watching!",
+            'thank you for watching!',
+            "so",
+            "beep!"
+            # "Thank you"  # this one can also be valid!!
+        ]
+        hallucinations = self.config.get("hallucination_list", default_hallucinations) \
+            if self.config.get("filter_hallucinations", True) else []
```

Comment:
@j1nx @goldyfruit @builderjer what do you think about this? is it a good thing to enable by default or should i make it False unless changed by user?

the list above was made from just saying wake word and not asking anything afterwards, i sometimes also get a "please subscribe" but far less common

Reply:
Maybe adding "Did you say something?" or "Not sure I heard you" could be nice.

Reply:
I mean the filtering of hallucinations

a bus message is emitted that a skill could listen for an speak those notifications if desired `"recognizer_loop:speech.recognition.unknown"`